modules/network-observability-ebpf-agent-alert.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+// * network_observability/network-observability-operator-monitoring.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-netobserv-dashboard-ebpf-agent-alerts_{context}"]
+= Using the eBPF agent alert
+
+When the Network Observability eBPF agent hashmap table is full, the eBPF agent processes flows at a degraded performance. When this is the case, an alert shows `NetObservAgentFlowsDropped`. If you see this alert, consider increasing the `cacheMaxFlows` in the `FlowCollector`, as shown in the following example.
+
+[NOTE]
+====
+Increasing the `cacheMaxFlows` may increase the memory usage of the eBPF agent. 
+====
+.Prerequisite
+* You must have eBPG metrics enabled by setting the `spec.agent.type.ebpf.metrics.enable` value to `true`. For more information, see "Enabling eBPF metrics".
+
+
+.Procedure
+
+. In the web console, navigate to *Operators* → *Installed Operators*.
+
+. Under the *Provided APIs* heading for the *Network Observability Operator*, select *Flow Collector*.
+
+. Select *cluster* then select the *YAML* tab.
+
+. Increase the `spec.agent.ebpf.cacheMaxFlows` value, as in the following YAML sample:
+[source,yaml]
+----
+apiVersion: flows.netobserv.io/v1beta2
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+  namespace: netobserv
+  deploymentModel: Direct
+  agent:
+    type: eBPF                                
+    ebpf:
+      cacheMaxFlows: 200000 <1>
+----
+<1> Increase the `cacheMaxFlows` value upwards from its value at the time of the `NetObservAgentFlowsDropped` alert.
```

Comment:
```suggestion
<1> Increase the `cacheMaxFlows` value from its value at the time of the `NetObservAgentFlowsDropped` alert.
```
I think "increase" implies upward so not needed

Reply:
Yep increase and upward are redundant 🙃